sql/branch.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TABLE emails DROP CONSTRAINT emails_participant_fkey;
+
+ALTER TABLE emails ALTER COLUMN participant TYPE bigint USING participant::bigint;
```

Comment:
The `participants.id` datatype is `bigserial`. So, the datatype of `emails.participant` need to match with this. In order to do so, we need to first convert it into `bigint` and then create a sequence for the column. We can read more about serial type from [here](http://www.postgresql.org/docs/current/static/datatype-numeric.html#DATATYPE-SERIAL).